orangecontrib/imageanalytics/widgets/owimageimport.py

```diff
@@ -695,20 +695,15 @@ def __init__(self, startdir, formats=DefaultFormats, report_progress=None):
 
     def run(self):
         imgmeta = []
-        filescanner = scan(self.startdir)
         patterns = ["*.{}".format(fmt.lower()) for fmt in self.formats]
-
-        def fnmatch_any(fname, patterns):
-            return any(fnmatch.fnmatch(fname.lower(), pattern)
-                       for pattern in patterns)
+        filescanner = scan(self.startdir, include_patterns=patterns)
 
```

Comment:
the `fnmatch_any` forced case insensitive matching (in combination with `fmt.lower()` few lines up). This is not retained here.

Reply:
Now `scan` no longer does what it claims.

```sh
touch foo.BAR
```
```python
next(iter(scan("./", include_patterns=["*.BAR"])))
```

Can you add a `case_insensitive` parameter to scan
```
case_insensitive : bool
   Ignore case in both filename(s) and pattern(s).